### PR TITLE
Class 02 Assignment

### PR DESCRIPTION
**Pull request by PeterBreen**
This took me about 3 hours. I spent more of the time doing some CSS cleanup and experimenting with design elements than jQuery, but probably not that much more - maybe a 60/40 split?  

While working with jQuery I decided to rename several variables in my js files - I did reuse a lot of the jquery from today's pair programming - but by renaming variables it meant I actually understood what was going on with the code better than I did after the pair exercise.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,37 +11,58 @@
...
-      <a href="mailto:peter@peterbreen.info">peter@peterbreen.info</p>
-      <ul class="social-list">
-        <li><a class="icon-github" href="https://github.com/PeterBreen"></a></li>
-        <li><a class="icon-linkedin" href="https://www.linkedin.com/in/peter-breen-96404142"></a></li>
-        <li><a class="icon-flickr3" href="https://www.flickr.com/photos/notpsion"></a></li>
-        <li><a class="icon-wordpress" href="http://peterbreen.info/wp"></a></li>
-      </ul>
-	   </section>
+    <header>
+      <h1 class="header-h1" id="top">Peter Breen</h1>
```
Comment (amyeleek) [inline index.html]: Why does this need a class? Do you have more than one H1?

File: index.html [hunk 2]
```diff
@@ -11,37 +11,58 @@
...
+           <li><a class="icon-flickr3" href="https://www.flickr.com/photos/notpsion"></a></li>
+           <li><a class="icon-wordpress" href="http://peterbreen.info/wp"></a></li>
+         </ul>
+       </section>
+      </main>
+      <footer>
+        <!-- TODO: footer content -->
+        <span>&copy; 2016 Peter Breen</span>
+      </footer>
+    <script src="vendor/jquery.min.js"></script> <!-- TODO: make a CDN link when wifi is more reliable - v2.2.2 -->
```
Comment (amyeleek) [inline index.html]: wifi will never become more reliable, wait until you can deploy to Heroku.

File: scripts/portfolio.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+var portfolioArticles = [];
+
+function Article (opts) {
```
Comment (amyeleek) [inline scripts/portfolio.js]: Article? Are you writing articles here?

**Comment by PeterBreen** [comment]
Made 3 commits to fix issues (thanks @amyeleek :+1: )

**Pull request merged**